src/obb_rules/auto_deploys/common.cljc

```diff
@@ -51,7 +51,7 @@
   "Builds a deploy action command"
   [player row idx [u q]]
   (let [coordinate (translator/coordinate player [(+ idx 1) row])]
-    [:deploy (int q) (unit/unit-name u) coordinate]))
+    [:deploy (int q) (keyword (unit/unit-name u)) coordinate]))
```

Comment:
This is a general comment and it is not intended for you to change anything. One thing that I noticed is that both strings and keywords are supported for the most relevant data, and the logic to support that is scattered throughout the codebase.
I can't help thinking that we should stick to one. I assume this is mostly because the deserialization from JSON generates strings and we need to be ready for that, right? If that is the case, I'd rather have a translation layer that properly converts all relevant things to keywords and make all remaining namespaces rely on that assumption.
The main advantages are that we would have all that logic gathered in one place, making it more explicit and reusable, while still simplifying the rest of the code.